src/docfx/apply-template/ApplyTemplate.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using HtmlReaderWriter;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class ApplyTemplate
+    {
+        public static bool Run(CommandLineOptions options)
+        {
+            using var errors = new ErrorWriter(options.Log);
+            if (!string.IsNullOrEmpty(options.Template))
+            {
+                if (string.IsNullOrEmpty(options.Directory))
+                {
+                    errors.Add(Errors.ApplyTemplate.StructuredJsonDirNotSpecified());
```

Comment:
Use `options.WorkingDirectory` to use current working directory when a user does not specify an argument.